### ISSUE TITLE
LF instead of CRLF?

### ISSUE THREAD

**Issue by masters3d** [Azure/azure-libraries-for-swift#10]
How much of this codebase is being generated? Whats the possibility to generate with LF instead of windows line ending? Some of the Swift tooling is not able to handle CRLF correctly yet. https://github.com/realm/SwiftLint/issues/1786

**Comment by masters3d** [comment]
Where I can find the code that generates this? I'd be interesting in applying some fixes to spacing that would me the code base consistent:

https://github.com/Azure/azure-libraries-for-swift/commit/dba968e684dbb0caa46633ecb4bc2db1c39452ef

**Comment by alvadb** [comment]
Hi, I apologize for the delayed response.  All the code in this library is auto generated except for the tests.  The repo for the generator is at https://github.com/Azure/autorest.swift.  I will look into this early next week if this is still outstanding.

**Comment by masters3d** [comment]
I've been trying to figure out how to generate this: https://github.com/Azure/autorest.modeler/pull/54#event-1555737268

Do you have a lists of steps I could follow? I'd love like to help.

**Comment by masters3d** [comment]
Yes. My end goal is to contribute back to the swift related tools. 

I got to run the solution in windows but I was missing how to get the swagger files.

**Comment by alvadb** [comment]
Awesome, that is great.  So you were able to build the swift extension and now you are ready to use it in autorest?  If so, the following are some examples on how to run the extension on a swagger file.

using README:
autorest https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current/specification/storage/resource-manager/readme.md --use=/Users/someuser/Documents/GitHub/autorest.swift --swift --tag=package-2017-06 --swift.output-folder=/Users/someuser/Documents/swift/autogened

--using input file:
autorest --input-file=https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/storage/resource-manager/Microsoft.Storage/stable/2017-10-01/storage.json --use=/Users/someuser/Documents/GitHub/autorest.swift --swift --swift.output-folder=/Users/someuser/Documents/swift/autogened
 
The Azure swagger files can be found in this repo: https://github.com/Azure/azure-rest-api-specs

**Comment by masters3d** [comment]
ah perfect! Now I understand this file! https://github.com/Azure/azure-libraries-for-swift/blob/master/gulpfile.js
